build.gradle

```diff
@@ -44,11 +44,11 @@ dependencies {
     play "net.logstash.logback:logstash-logback-encoder:4.11"
 
     play "com.netaporter:scala-uri_$scalaVersion:0.4.16"
-    play "net.codingwell:scala-guice_$scalaVersion:4.2.1"
+    play "net.codingwell:scala-guice_\$scalaVersion:4.2.1"
```

Comment:
Using the scaping `\` here is incorrect:

https://travis-ci.org/playframework/play-scala-rest-api-example/jobs/476876598#L653-L658

It must only be used in our seeds since the seeds also consider `$scalaVersion` like variables that to replace.

Reply:
so how, I lost track of this PR. I guess this is happening because we changed it in template control to fix the seeds. 

We need to find a better way of doing this.